File: src/main/java/nu/peg/slack/pt/api/slack/model/ConnectionDetailAttachment.java
package nu.peg.slack.pt.api.slack.model;

import java.util.List;

import lombok.*;

@Data
@AllArgsConstructor
@EqualsAndHashCode
public class ConnectionDetailAttachment implements Attachment {
    private String pretext, color;
    private List<AttachmentField> fields;
}

File: src/main/kotlin/nu/peg/slack/pt/api/slack/model/ConnectionDetailAttachment.kt
package nu.peg.slack.pt.api.slack.model

import lombok.*

@Data
@AllArgsConstructor
@EqualsAndHashCode
class ConnectionDetailAttachment : Attachment {
    private val pretext: String? = null
    private val color: String? = null
    private val fields: List<AttachmentField>? = null
}
